app/src/main/java/com/everest/moviedb/MovieTabAdapter.kt

```diff
@@ -0,0 +1,27 @@
+package com.everest.moviedb
+
+import android.content.Context
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+
+class MovieTabAdapter(fragmentManager: FragmentManager, private val context: Context) :
+    FragmentStatePagerAdapter(fragmentManager) {
+    val categories = listOf(
```

Comment:
you can just pass this list right, Because we have to pass Context for this single purpose